weather_dl/download_pipeline/util.py

```diff
@@ -134,9 +135,18 @@ def to_json_serializable_type(value: t.Any) -> t.Any:
     return value
 
 
-def fetch_geo_polygon(area: list) -> str:
+def fetch_geo_polygon(area: t.Union[list, str]) -> str:
     """Calculates a geography polygon from an input area."""
-    n, w, s, e = area
+    # Ref: https://confluence.ecmwf.int/pages/viewpage.action?pageId=151520973
```

Comment:
TIL about `'E'`! One minor nit: Can you add a quick note in the comment that this area refers to Europe?